src/me/mrCookieSlime/Slimefun/Objects/SlimefunItem/machines/ElectricDustWasher.java

```diff
@@ -101,6 +108,7 @@ else if (ChargableBlock.isChargable(b)) {
 					else if (SlimefunStartup.chance(100, 25)) adding = SlimefunItems.SILVER_DUST;
 					
 					MachineRecipe r = new MachineRecipe(4 / getSpeed(), new ItemStack[0], new ItemStack[] {adding});
+					if (!fits(b, r.getOutput())) return;
```

Comment:
This should probably only be called if legacy_dust_washer is true.

Aka (legacy_dust_washer && fits...)